src/webhooks/options-automator/options-automator.ts

```diff
@@ -76,6 +76,10 @@ export async function sendOptionAutomatorUpdatesToDataDog(
 }
 
 export async function messageSlack(message: OptionsAutomatorResponse) {
+  if (message.source !== 'options-automator') {
```

Comment:
Why is the payload being sent here, if it does not pertain to `options-automator`?

Reply:
We want to send to DD, but not slack?

Reply:
if the way the option was changed wasn't by the options-auto (the other way is by the admin tool) then I dont want it to go to slack, since the channel is specifically for options-automator changes only